### ISSUE TITLE
"Go Time" no longer current on Google Podcasts

### ISSUE THREAD

**Issue by DazWilkin** [thechangelog/changelog.com#410]
I noticed recently that I'm no longer receiving Go Time podcasts on Google Podcasts.

The last podcast in my subscription is 15 July 2021 "SIV and the V2+ issue"

Is this my error? I tried unsubscribing and resubscribing but this makes no difference.

![image](https://user-images.githubusercontent.com/1022400/161473144-17cd813d-2b9c-47ed-a744-29c57cdd3003.png)

**Comment by jerodsanto** [comment]
Yeah [our feeds are all jacked up on Google Podcasts](https://twitter.com/changelog/status/1510266542398877703). A support request has been filed, so we'll see what happens from here.

(I suggest using this opportunity to find a cool, indie podcast app and give them your business.)

**Comment by DazWilkin** [comment]
Thanks for your response.

I've filed feedback with Google too 

I'd prefer to continue to use Google Podcasts.
